docs/features/multiple-networks/multi-homing.md

```diff
@@ -111,17 +178,19 @@ spec:
   These IPs will be removed from the assignable IP pool, and never handed over
   to the pods.
 - `allowPersistentIPs` (boolean, optional): persist the OVN Kubernetes assigned
-   IP addresses in a `ipamclaims.k8s.cni.cncf.io` object. This IP addresses will
-   be reused by other pods if requested. Useful for KubeVirt VMs. Only makes
-   sense if the `subnets` attribute is also defined.
+  IP addresses in a `ipamclaims.k8s.cni.cncf.io` object. This IP addresses will
+  be reused by other pods if requested. Useful for KubeVirt VMs. Only makes
+  sense if the `subnets` attribute is also defined.
 
 **NOTE**
+
 - when the subnets attribute is omitted, the logical switch implementing the
   network will only provide layer 2 communication, and the users must configure
   IPs for the pods. Port security will only prevent MAC spoofing.
-- switched - layer2 - secondary networks **only** allow for east/west traffic.
+- switched - layer2 - secondary-networks **only** allow for east/west traffic.
```

Comment:
why are you adding a dash here ?

Reply:
Wanted to keep things consistent, but nothing wrong with keeping it consistent to `secondary networks`. Will change

Reply:
DONE